src/timeline/timeline-item.scss

```
@@ -0,0 +1,84 @@
+.TimelineItem {
+  position: relative;
+  display: flex;
+  padding: $spacer-3 0;
+  margin-left: $spacer-3;
+
+  // The Timeline
+  &::before {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    display: block;
+    width: 2px;
+    content: "";
+    background-color: $gray-200;
+  }
+
+  &:target .TimelineItem-badge {
+    border-color: $blue-400;
+    box-shadow: 0 0 0.2em $blue-200;
+  }
+}
+
+.TimelineItem-badge {
+  position: relative;
+  z-index: 1;
+  display: flex;
+  width: $spacer-5;
+  height: $spacer-5;
+  margin-right: $spacer-2;
+  margin-left: -$spacer-3 + 1;
+  color: $gray-700;
+  align-items: center;
+  background-color: $gray-200;
+  border: 2px solid $white;
+  border-radius: 50%;
+  justify-content: center;
+  flex-shrink: 0;
+}
+
+.TimelineItem-body {
```

This was added to the component in https://github.com/github/github/pull/125487

We're trying to combat items expanding larger than their containers when content has long line lengths. https://dfmcphee.com/flex-items-and-min-width-0/ is related reading.

```suggestion
.TimelineItem-body {
  min-width: 0;
```